ci/jenkins/checkout.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+set -e
+
+tesbted_name="$1"
+force="$2"
+
+if [ -z "${tesbted_name}" ]; then
+  echo "Usage: $0 <testbed_name> [-f]"
+  exit 1
+fi
+
+if [ ! -e "../terraform.tfstate.d/current/${tesbted_name}" ]; then
+  echo "${tesbted_name} does not exist in remote workspace"
+  exit 1
+fi
+
+if [ -e "terraform.tfstate.d/${tesbted_name}" ]; then
```

Comment:
Terraform already has concept of workspaces not sure if we want to recreate another wrapper around terraform,

https://www.terraform.io/language/state/workspaces